tests/test_schema.py

```diff
@@ -99,35 +99,19 @@ class Person(models.Base):
 def test_model_with_constructors():
 
     class Car(models.Base):
-
-        def __init__(self, some_value):
-            pass
-
         brand = fields.StringField(required=True)
         registration = fields.StringField(required=True)
 
     class Toy(models.Base):
-
-        def __init__(self, some_value):
-            pass
-
         name = fields.StringField(required=True)
 
     class Kid(models.Base):
-
-        def __init__(self, some_value):
-            pass
-
         name = fields.StringField(required=True)
         surname = fields.StringField(required=True)
         age = fields.IntField()
         toys = fields.ListField(Toy)
 
     class Person(models.Base):
-
-        def __init__(self, some_value):
```

Comment:
Why are you removing inits? They should stay, otherwise this test doesn't make sense. Besides they should work anyway, default feature is backwad compatible from what I see here

Reply:
I misunderstood this test, and I still not sure if I understand: You never create a `Person` instance, so how does the class having a constructor matters in any way?

Reply:
@beregond please advise on what do you want to be done here.
I can revert my changes, but still I think this test is meaningless.
There's nothing special about models with constructors when it comes to schema generation.
Moreover, the constructor is never called during the test, nor does it affect it in any way.

IMO this test should be deleted.
Waiting for your decision on this one.

Reply:
@avrahamshukron please let it be, first its not the part of feature, second - it is intended to fail the moment some part of package starts to require instances instead of classes. Actually more suitable here would be to put `raise NotImplemented` in each of those inits :)